RouteToDo/Place.m

```diff
@@ -26,7 +26,11 @@ - (instancetype) initWithDictionary:(NSDictionary *)dictionary {
 
     place.name = dictionary[@"name"];
     place.fullDescription = dictionary[@"full_description"];
-    place.coordinates = CLLocationCoordinate2DMake([dictionary[@"coordinates"][@"latitude"] doubleValue], [dictionary[@"coordinates"][@"longitude"] doubleValue]);
+    if (dictionary[@"coordinates"] != nil) {
```

Comment:
this isn't ideal but i didn't want to break other things already using "coordinates"